### ISSUE TITLE
BC break switching to laminas/laminas-diagnostics

### ISSUE THREAD

**Issue by ndench** [liip/LiipMonitorBundle#231]
When [switching to Laminas Diagnostics](https://github.com/liip/LiipMonitorBundle/commit/4d32cd910957e05f0028ede686bc71168b3c4422#diff-c96cc9f9dca6efb17ee0b90299287a9d) there was a BC break for users who are defining custom reporters and extending the built in reporters. To fix the BC break, users merely need to update the their import statements:

```diff
- use ZendDiagnostics\Result\Collection as ResultsCollection;
+ use Laminas\Diagnostics\Result\Collection as ResultsCollection;
```

**Comment by kbond** [comment]
Damn.. the https://github.com/laminas/laminas-zendframework-bridge doesn't alias this?

**Comment by ndench** [comment]
Hmm yeah, that didn't seem to work. I'm not sure why though.
After the composer update it got through our CI process fine but once it was deployed, the first time we hit the monitor endpoint for a health check we got an exception.

**Comment by adonix** [comment]
This bundle uses the `security checker` from [sensioLabs](https://github.com/sensiolabs/security-checker), which is a dependency in the **laminas-diagnostics** bundle, however, the [security-checker](https://security.symfony.com) is no longer supported. Is there a fix for this? It is being recommended to use [this](https://github.com/fabpot/local-php-security-checker) an alternative. Anyone had luck with this?

**Comment by kbond** [comment]
Yep, this is being worked on: https://github.com/liip/LiipMonitorBundle/pull/258

**Comment by adonix** [comment]
@kbond thanks for the update 👍
